docs/research/geode_lx.md

```diff
@@ -0,0 +1,206 @@
+ALIX and AMD Geode LX
+=====================
+
+
+This paper contains research about Geode LX CAR and whole effort to implement
+`EARLY_CBMEM_INIT` in coreboot for PC Engines ALIX platforms.
+
+Table of Contents
+=================
+
+
```

Comment:
@miczyg1 same here, TOC should probably be with different header then title, maybe

header
---------

Reply:
Fixed